readme.md

```diff
@@ -14,6 +14,7 @@ Utility functions, classes, types, and scripts in uncompiled TS, for Deno.
 - [`io`](#io)
 - [`json`](#json)
 - [`md`](#md)
+- ['mermaid'](#mermaid)
   - [`script/evalCodeBlocks`](#scriptevalcodeblocks)
```

Comment:
Ordering issue. I recommend installing the Markdown All-In-One VS Code extension which automatically maintains this table of contents on save.